### ISSUE TITLE
Support collaborate place picking

### ISSUE THREAD

**Issue by mhall119** [GetTogetherComm/GetTogether#61]
Make it easier to pick a good location for a get together based on the locations and preferences of the people planning on attending

**Comment by craigmaloney** [comment]
May also be nice to have a list of previously used places per team. I know I'm having to re-create new places because there's no way to search on old places (at least I'm not aware of a way).

**Comment by mhall119** [comment]
Yup, that's already in issue #49 and targeted for the next milestone 0.4.0
